community/brillo/depends

```diff
@@ -0,0 +1 @@
+go make
```

Comment:
The depends file can be removed. With the build file changes no dependencies are needed.

Reply:
dependency removed and new commits pushed.